modules/apps/account/account-admin-web/src/main/java/com/liferay/account/admin/web/internal/display/context/ViewAccountGroupsManagementToolbarDisplayContext.java

```diff
@@ -48,6 +55,40 @@ public ViewAccountGroupsManagementToolbarDisplayContext(
 			searchContainer);
 	}
 
+	@Override
+	public List<DropdownItem> getActionDropdownItems() {
+		return DropdownItemList.of(
+			() -> {
+				DropdownItem dropdownItem = new DropdownItem();
+
+				dropdownItem.putData("action", "deleteAccountGroups");
+
+				PortletURL deleteAccountGroupsURL =
+					liferayPortletResponse.createActionURL();
+
+				deleteAccountGroupsURL.setParameter(
+					ActionRequest.ACTION_NAME,
+					"/account_groups_admin/delete_account_groups");
+
+				dropdownItem.putData(
+					"deleteAccountGroupsURL",
+					deleteAccountGroupsURL.toString());
+
+				dropdownItem.setIcon("times-circle");
+				dropdownItem.setLabel(LanguageUtil.get(request, "delete"));
+				dropdownItem.setQuickAction(true);
+
+				return dropdownItem;
+			});
+	}
+
+	public List<String> getAvailableActions(
+			AccountGroupDisplay accountGroupDisplay)
+		throws PortalException {
+
+		return ListUtil.fromArray("deleteAccountGroups");
+	}
+
```

Comment:
This is not needed unless there's condition for making the delete action available (e.g. permission checks)